### PR TITLE
[CloudMinds] 10320-CCU

### PR DESCRIPTION
**Pull request by Kenny-SZ**
This LWM2M Object includes CCU(Central Control Unit) information and the link to CCU APPs.

### CODE REVIEW

**Comment by seanmcilroy29** [comment]
@Kenny-SZ - No Object file has been submitted to this PR

**Comment by Kenny-SZ** [comment]
> @Kenny-SZ - No Object file has been submitted to this PR

Repushed the file.

**Comment by seanmcilroy29** [comment]
@Kenny-SZ Object now visible

File: 10320.xml [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+        <Description><![CDATA[The software version of the CCU.]]></Description>
+      </Item>
+	  <Item ID="4">
+        <Name>CCU Memory Size</Name>
+        <Operations>R</Operations>
+        <MultipleInstances>Single</MultipleInstances>
+        <Mandatory>Mandatory</Mandatory>
+        <Type>Integer</Type>
+        <RangeEnumeration/>
+        <Units>G</Units>
```
Comment (seanmcilroy29) [inline 10320.xml]: Incorrect unit applied

File: 10320.xml [hunk 2]
```diff
@@ -0,0 +1,153 @@
...
+        <Description><![CDATA[Total storage of the CCU.]]></Description>
+      </Item>
+	  <Item ID="6">
+        <Name>CCU Available Storage</Name>
+        <Operations>R</Operations>
+        <MultipleInstances>Single</MultipleInstances>
+        <Mandatory>Mandatory</Mandatory>
+        <Type>Integer</Type>
+        <RangeEnumeration/>
+        <Units>M</Units>
```
Comment (seanmcilroy29) [inline 10320.xml]: Incorrect unit applied
If M is for meter, SenML defines this as ``m``

File: 10320.xml [hunk 3]
```diff
@@ -0,0 +1,153 @@
...
+        <Description><![CDATA[Total Memory Size of the CCU (expressed in gigabyte).]]></Description>
+      </Item>	  
+	  <Item ID="5">
+        <Name>CCU Storage</Name>
+        <Operations>R</Operations>
+        <MultipleInstances>Single</MultipleInstances>
+        <Mandatory>Mandatory</Mandatory>
+        <Type>Integer</Type>
+        <RangeEnumeration/>
+        <Units>M</Units>
```
Comment (seanmcilroy29) [inline 10320.xml]: Incorrect unit applied
If M is for meter, SenML defines this as ``m``

File: 10320.xml (hunk 1 above)
Comment (Kenny-SZ) [reply 10320.xml]: Changed from G to GB.

File: 10320.xml (hunk 2 above)
Comment (Kenny-SZ) [reply 10320.xml]: Changed from M to GB.

File: 10320.xml (hunk 3 above)
Comment (Kenny-SZ) [reply 10320.xml]: Changed from M to GB.

File: 10320.xml (hunk 1 above)
Comment (Kenny-SZ) [reply 10320.xml]: GB is the storage unit, Gigabyte.

File: 10320.xml (hunk 2 above)
Comment (Kenny-SZ) [reply 10320.xml]: GB is the storage unit, Gigabyte.

File: 10320.xml (hunk 3 above)
Comment (Kenny-SZ) [reply 10320.xml]: GB is the storage unit, Gigabyte.

**Comment by Kenny-SZ** [comment]
Changed GB to gigaB.

**Comment by Kenny-SZ** [comment]
Changed gigaB to GB.

**Pull request merged**